pylibui/controls/box.py

```diff
@@ -34,6 +26,35 @@ def append(self, child, stretchy=0):
         :return: None
         """
         libui.uiBoxAppend(self.control, child.pointer(), stretchy)
+        self.children.append(child)
+
+    def delete(self, index):
+        """
+        Deletes a child from a box.
+
+        :param index: int
+        :return: None
+        """
+        libui.uiBoxDelete(self.control, index)
+        self.children[index].destroy()
+        del self.children[index]
+
+    def getPadded(self):
+        """
+        Returns whether the box is padded.
```

Comment:
If it returns an int, is it returning the padding int value of the box? The "whether" word implies a boolean.